src/argilla/server/apis/v0/handlers/users.py

```diff
@@ -20,7 +20,7 @@
 from pydantic import parse_obj_as
 from sqlalchemy.orm import Session
 
-from argilla.server import models
+from argilla.server import database, models
```

Comment:
You don't need to import `database` we are importing `get_db` later:

```suggestion
from argilla.server import models
```